packages/arb-validator-core/ethbridge/arbRollup.go

```diff
@@ -47,24 +47,26 @@ func newRollup(address ethcommon.Address, client ethutils.EthClient, auth *Trans
 func (vm *arbRollup) PlaceStake(ctx context.Context, stakeAmount *big.Int, proof1 []common.Hash, proof2 []common.Hash) ([]arbbridge.Event, error) {
 	vm.auth.Lock()
 	defer vm.auth.Unlock()
-	call := &bind.TransactOpts{
-		From:    vm.auth.auth.From,
-		Signer:  vm.auth.auth.Signer,
-		Context: ctx,
-	}
-	blankAddress := ethcommon.Address{}
-	st, err := vm.ArbRollup.GetStakeToken(&bind.CallOpts{Context: ctx})
-	if err != nil {
-		return nil, err
-	}
-	if st == blankAddress {
-		call.Value = stakeAmount
-	}
-	tx, err := vm.ArbRollup.PlaceStake(
-		call,
-		common.HashSliceToRaw(proof1),
-		common.HashSliceToRaw(proof2),
-	)
+	tx, err := vm.auth.makeTx(ctx, func(auth *bind.TransactOpts) (*types.Transaction, error) {
+		call := &bind.TransactOpts{
+			From:    auth.From,
+			Signer:  auth.Signer,
+			Context: ctx,
+		}
```

Comment:
I think we want to just use `auth` instead of building `call` here

Reply:
sounds good